.golangci.yml

```diff
@@ -78,7 +79,6 @@ linters:
     - unconvert
     # - unparam
     - unused
-    - varcheck
```

Comment:
Regarding the 3 removed checks: the linter warned they appear not to be maintained anymore and that their functionality can be found in `unused`.